### DISCUSSION TITLE
clairctl config.yaml howto

### DISCUSSION THREAD

**Discussion by flomickl** [quay/clair#2095] [Q&A]
Hi,

clairctl has a global option for passing a configuration. But I don't really understand the use case and can't find any syntax and possible parameters I could use here. Assuming that it is not the clair config.yaml itself

```
--config value, -c value     clair configuration file (default: "config.yaml") [$CLAIR_CONF]
```

Is this 1:1 the clair config file? Is there any use case why I pass the clair conf.yaml here again?

In my understanding, I would only pass additional information like host , issuer and key and maybe skip verify or something like this here with the goal to connect with clair and nothing else.
But then I can't find any information about possible parameters.

So what parameters and syntax does this yaml config have to achieve my goal?
Many thanks in advance 

Many thanks.

**Comment by hdonnay** [comment]
> Is this 1:1 the clair config file?

Yes.

> Is there any use case why I pass the clair conf.yaml here again?

For when some parts of the configuration are needed, like authentication and `import-updaters`/`export-updaters`.
